sim/midas/targetutils/src/main/scala/midas/annotations.scala

```diff
@@ -202,10 +202,15 @@ case class AutoCounterCoverModuleAnnotation(target: String) extends ChiselAnnota
 
 object PerfCounter {
   /**
-    * Annotates a Bool representing a target event (ex. L1 D$ miss)  that
-    * should be tracked by AutoCounter
+    * Labels a signal as an event for which an host-side counter (an
+    * "AutoCounter") should be generated).  Events can be multi-bit to encode
+    * multiple occurances in a cycle (e.g., the number of instructions retired
+    * in a superscalar processor). NB: Golden Gate will not generate the
+    * coutner unless AutoCounter is enabled in your the platform config. See
+    * the docs for more info.
```

Comment:
Should this be a ref to a specific point in the docs or is that too much to maintain?